openapi/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.8"
+
+services:
+  swagger-cli:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: swagger-cli
+    volumes:
+      - ./:/openapi
+    command: "bundle -o /openapi/mergedOpenApi.json /openapi/versions/v1.yml"
+    restart: "no"
+
+  swagger:
+    image: swaggerapi/swagger-ui
+    container_name: swagger
+    environment:
+      - SWAGGER_JSON=/openapi/mergedOpenApi.json
+    ports:
+      - "8083:8080"
+    volumes:
+      - ./:/openapi
+    depends_on:
+      - swagger-cli
+    restart: always
+
+  redoc:
```

Comment:
This is unnecessary

Reply:
removed it. updated in 7b25eb673cca15ddae3e924df99684249951f607